constants.py

```diff
@@ -49,25 +50,27 @@
         "s390x-ubuntu-2204-deb-autobake",
         "s390x-rhel-8",
         "s390x-rhel-8-rpm-autobake",
+        "s390x-rhel-9",
+        "s390x-rhel-9-rpm-autobake",
         "s390x-sles-15",
         "s390x-sles-15-rpm-autobake",
         ]
 
-builders_autobake=["amd64-centos-7-rpm-autobake", "amd64-centos-8-rpm-autobake", "amd64-debian-9-deb-autobake", "x86-debian-9-deb-autobake", "amd64-debian-10-deb-autobake", "amd64-debian-11-deb-autobake", "amd64-debian-sid-deb-autobake", "x86-debian-sid-deb-autobake", "amd64-fedora-34-rpm-autobake", "amd64-fedora-35-rpm-autobake", "amd64-fedora-36-rpm-autobake", "amd64-rhel-7-rpm-autobake", "amd64-rhel-8-rpm-autobake", "amd64-opensuse-15-rpm-autobake", "amd64-sles-12-rpm-autobake", "amd64-sles-15-rpm-autobake", "amd64-ubuntu-1804-deb-autobake", "amd64-ubuntu-2004-deb-autobake", "amd64-ubuntu-2110-deb-autobake", "amd64-ubuntu-2204-deb-autobake", "aarch64-ubuntu-1804-deb-autobake", "aarch64-ubuntu-2004-deb-autobake", "aarch64-ubuntu-2204-deb-autobake", "aarch64-ubuntu-2110-deb-autobake", "ppc64le-debian-9-deb-autobake", "ppc64le-debian-10-deb-autobake", "ppc64le-debian-11-deb-autobake", "ppc64le-debian-sid-deb-autobake", "ppc64le-ubuntu-1804-deb-autobake", "ppc64le-ubuntu-2004-deb-autobake", "ppc64le-ubuntu-2110-deb-autobake", "ppc64le-ubuntu-2204-deb-autobake", "ppc64le-centos-7-rpm-autobake", "ppc64le-rhel-7-rpm-autobake", "ppc64le-rhel-8-rpm-autobake", "aarch64-debian-10-deb-autobake", "aarch64-debian-11-deb-autobake", "aarch64-debian-sid-deb-autobake", "aarch64-debian-9-deb-autobake", "aarch64-fedora-34-rpm-autobake", "aarch64-fedora-35-rpm-autobake", "aarch64-fedora-36-rpm-autobake", "aarch64-centos-7-rpm-autobake", "aarch64-centos-8-rpm-autobake", "aarch64-rhel-7-rpm-autobake", "aarch64-rhel-8-rpm-autobake", "s390x-ubuntu-2004-deb-autobake", "s390x-ubuntu-2204-deb-autobake", "s390x-rhel-8-rpm-autobake", "s390x-sles-15-rpm-autobake"]
-
 builders_big=["amd64-ubuntu-1804-bigtest"]
 
 # Generate install builders based on the os_info data
 builders_install = []
 builders_upgrade = []
+builders_autobake = []
 for os in os_info:
     for arch in os_info[os]['arch']:
-        builder_name_install = arch + '-' + os + '-' + os_info[os]['type'] + '-autobake-install'
-        builders_install.append(builder_name_install)
-        builder_name_minor_upgrade = arch + '-' + os + '-' + os_info[os]['type'] + '-autobake-minor-upgrade'
-        builders_upgrade.append(builder_name_minor_upgrade)
-        builder_name_major_upgrade = arch + '-' + os + '-' + os_info[os]['type'] + '-autobake-major-upgrade'
-        builders_upgrade.append(builder_name_major_upgrade)
+        builder_name_autobake = arch + '-' + os + '-' + os_info[os]['type'] + '-autobake'
+        builders_autobake.append(builder_name_autobake)
```

Comment:
Looks good! The only thing that I noticed missing is updating `os_info,yaml` with the new additions and probably double check that all entries are there

Reply:
fc37 isn't there though rhel9 and centos-stream9 are. s390x workers are waiting on a libvirt VM.

So I'm thinking and hoping this is all ready now.